components/feature/prompts/src/main/java/mozilla/components/feature/prompts/TimePickerDialogFragment.kt

```diff
@@ -78,6 +87,22 @@ internal class TimePickerDialogFragment : PromptDialogFragment(), DatePicker.OnD
                     it.setButton(BUTTON_POSITIVE, context.getString(R.string.mozac_feature_prompts_set_date), this)
                     it.setButton(BUTTON_NEGATIVE, context.getString(R.string.mozac_feature_prompts_cancel), this)
                 }
+            SELECTION_TYPE_MONTH -> AlertDialog.Builder(context)
+                .setTitle(R.string.mozac_feature_prompts_set_month)
+                .setView(inflateDateMonthPicker(LayoutInflater.from(context)))
+                .create()
+                .also {
+                    it.setButton(
```

Comment:
nit: spacing should be the same as above
```suggestion
                    it.setButton(BUTTON_POSITIVE, context.getString(R.string.mozac_feature_prompts_set_date), this)
```

Reply:
Thanks my formatted did that 😄

Reply:
Fixed!